modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/objectDefinitionUtil.ts

```diff
@@ -51,6 +55,127 @@ export async function deleteObjectDefinition(
 	});
 }
 
+export async function getDeleteObjectDefinition(
```

Comment:
@igor-franca I think we can remove the "get" in the beginning of the function name.